app/views/books/_form.haml

```diff
@@ -6,11 +11,42 @@
         - @book.errors.full_messages.each do |message|
           %li= message
 
-  = f.label 'Cover Image'
-  = f.file_field :cover_image
+  .question
+    = f.label 'image', 'Cover Image:'
+    = f.file_field :cover_image
+
+  .question
+    = f.label 'content', 'eBook PDF:'
+    = f.file_field :content
+
+  .question
+    = f.label 'title', 'Title:'
+    = f.text_field :title
+
+  .question
+    = f.label 'author', 'Author:'
+    = f.text_field :author
+
+  .question
+    = f.label 'edition', 'Edition:'
+    = f.number_field :edition, :value => 1, min: 1
```

Comment:
Better to use `value: 1` syntax vs `:value => 1`.

reference: https://launchschool.com/books/ruby/read/hashes